cursos/helm/00_solucions/99_other_wp/actividade-1-charts-helm/wp-release/templates/wp-deploy.yaml

```diff
@@ -0,0 +1,39 @@
+# wp-deploy.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: deploy-wp
+  labels:
+    app: "deploy-wp"
+    tipo: "practica"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: deploy-wp
+  template: 
+    metadata:
+      labels:
+        app: deploy-wp
+    spec:
+      containers:
+      - name: deploy-wp
+        image: wordpress:6.1
+        env:
+          - name: "MYSQL_PASSWORD"
+            valueFrom:
+              secretKeyRef:
+                name: wp-secrets
+                key: root-password
+        envFrom:
+          - configMapRef: 
+              name: wp-config
+        ports:
+        - containerPort: 8080
+        volumeMounts:
+        - name: wordpress-persistent-storage
+          mountPath: /var/www/html
+      volumes:
+      - name: wordpress-persistent-storage
+        persistentVolumeClaim:
+          claimName: wp-pv-claim
```

Comment:
This disk should be a shared storage, like aws efs or Azure Fileshare. To be able to mount it in multiple pod replicas